inc/composer/class-installer.php

```diff
@@ -10,10 +10,13 @@
 use Composer\Installers\Installer as BaseInstaller;
 use Composer\Package\PackageInterface;
 
+// @codingStandardsIgnoreStart WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase -- Snake case to match Composer.
+
 /**
  * Altis Core Composer Installer.
  */
 class Installer extends BaseInstaller {
+	protected $installOverrides = [];
```

Comment:
Thought the linter would require a docblock but fair enough!

Reply:
Hmm, think this might be due to an overzealous ignore then.